### ISSUE TITLE
peerDependency and shrinkwrap for angular version 4

### ISSUE THREAD

**Issue by tomalex0** [Gbuomprisco/ng2-tag-input#302]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**

Peer dependency still pointed to angular `2.4.0`, which prevents to create shrinkwrap
https://github.com/Gbuomprisco/ng2-tag-input/blob/master/package.json#L78

**Expected behavior**

Peer dependency should be same as latest angular version specified in dependency

**Minimal reproduction of the problem with instructions**

`npm install ng2-tag-input@1.0.1 --save`  with shrinkwrap enabled

**What is the motivation / use case for changing the behavior?**

**What do you use to build your app? (SystemJS, Webpack). Please specify the version**

* **Angular version:** 2.0.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by tomalex0** [comment]
Created a pulll request https://github.com/Gbuomprisco/ng2-tag-input/pull/303

**Comment by Gbuomprisco** [comment]
Hi @tomalex0,

thanks for the PR! will publish tomorrow

**Comment by Gbuomprisco** [comment]
Published ng2-tag-input@1.0.2 :)

**Comment by tomalex0** [comment]
thanks a lot @Gbuomprisco
